app/lib/backend_api_logic/proxy_extension.rb

```diff
@@ -5,24 +5,30 @@ module ProxyExtension
     extend ActiveSupport::Concern
 
     included do
-      delegate :backend_api, to: :service, allow_nil: true
-      delegate :private_endpoint, :private_endpoint=, to: :backend_api, allow_nil: true, prefix: true
-      delegate :default_api_backend, to: 'BackendApi'
+      delegate :api_backend,       to: :service
+      delegate :api_backend=,      to: :service
```

Comment:
I think these 2 lines can be done in 1 single delegate in either 1 line or 2 lines

Reply:
It was like that before but I do not know why @duduribeiro changed it ...